code/datums/emergency_calls/retired.dm

```diff
@@ -0,0 +1,55 @@
+/datum/emergency_call/retired
+	name = "Retired TGMC Veteran Squad"
+	base_probability = 15
+	alignement_factor = -1
+
+/datum/emergency_call/retired/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You are an old, retired member of the TerraGov marine corps.</b>")
+	to_chat(H, "<B>Althought you may be past your prime, high command has deemed you capable enough to be sent to the [SSmapping.configs[SHIP_MAP].map_name], which has recently sent out a distress signal.</b>")
+	to_chat(H, "<B>Investigate why the distress signal was sent and show the younger generation how it's done!</b>")
+
+/datum/emergency_call/retired/create_member(datum/mind/mind_to_assign)
+	. = ..()
+	if(!.)
+		return
+	var/mob/original = mind_to_assign.current
+	var/mob/living/carbon/human/H = .
+
+	if(H.gender == MALE)
+		H.h_style = pick("Bald", "Balding Hair", "Balding Fade", "Balding ponytail", "Balding medium")
+	else
+		H.h_style = pick("Overeye Very Short", "Updo", "Ponytail 1")
+	H.r_hair = 235
+	H.g_hair = 235
+	H.b_hair = 235
+	H.r_facial = 235
+	H.g_facial = 235
+	H.b_facial = 235
+	H.update_hair()
+
+	mind_to_assign.transfer_to(H, TRUE)
+	H.fully_replace_character_name(mind_to_assign.name, H.real_name)
+
+	if(original)
+		qdel(original)
+
+	print_backstory(H)
+
+	H.amputate_limb(BODY_ZONE_L_LEG)
```

Comment:
It would be cool if some members had other body mods (like bionic limbs, or a missing arm but a armour mounted gun etc), instead of literally everyone being in a chair.

Although it would probably be easier to put that sort of stuff in the pre/post equip stuff for a particular outfit variant.